app/presenters/waste_carriers_engine/certificate_presenter.rb

```diff
@@ -6,7 +6,7 @@ class CertificatePresenter < BasePresenter
     LOCALES_KEY = ".waste_carriers_engine.pdfs.certificate"
 
     def carrier_name
-      return company_name unless business_type == "soleTrader"
+      return company_name unless upper_tier? && business_type == "soleTrader"
```

Comment:
I do find reading an `unless` with `&&` very very hard!! Can we make it an `if` and/or make a method for the check only? 🙇‍♀

Reply:
Agreed, it is messy! I've tidied it up 👍

Reply:
And I love it now! 😍